.gitignore

```diff
@@ -202,3 +205,6 @@ dmypy.json
 # tmp
 tmp.py
 tmp.*
+testSheets.ipynb
+testSheets.ipynb
```

Comment:
duplicates. you don't really need to add this to git ignore. you can just selectively not add some files to a commit (which is often the way to go for temporary files

Reply:
resolved in new PR